tests/conftest.py

```diff
@@ -64,12 +64,20 @@ def all_versions(request):
     return version
 
 
+@pytest.fixture
+def latest_version():
+    global VERSIONS
+    return VERSIONS[0]
```

Comment:
stray fixture

Reply:
It's used here: https://github.com/vyperlang/vvm/blob/3aa628797bbbc538dfc4761573c671120087c2f6/tests/test_versioning.py#L45-L46